gc/base/OMRVMInterface.cpp

```diff
@@ -81,6 +81,15 @@ GC_OMRVMInterface::getOmrHookInterface(MM_GCExtensionsBase *extensions)
 }
 
 /**
+ * Return the OMR private hook interface for the Memory Manager
+ */
+J9HookInterface**
+GC_OMRVMInterface::getOmrPrivateHookInterface(MM_GCExtensionsBase *extensions)
+{
```

Comment:
Is this really what you want to do?  It exposes the private hook interface to everyone.... which means it isn't private anymore.